addons/hr_expense/tests/test_expenses.py

```diff
@@ -244,27 +244,20 @@ def test_partial_payment_multiexpense(self):
         payable_move_lines = exp_move_lines.filtered(lambda l: l.account_id.internal_type == 'payable')
         self.assertEqual(len(payable_move_lines), 2)
 
-        WizardRegister = self.env["hr.expense.sheet.register.payment.wizard"].with_context(
-            active_model=expense._name, active_id=expense.id, active_ids=expense.ids
-        )
-
-        register_pay1 = WizardRegister.create({
-            'journal_id': bank_journal.id,
-            'payment_method_id': outbound_pay_method.id,
-            'amount': 300,
-        })
-        register_pay1.expense_post_payment()
+        payment = self.env['account.payment.register']\
+            .with_context(active_model='account.move', active_ids=expense.account_move_id.ids)\
+            .create({'amount': 300.0})\
+            ._create_payments()
 
         exp_move_lines = expense.account_move_id.line_ids
         payable_move_lines = exp_move_lines.filtered(lambda l: l.account_id.internal_type == 'payable')
         self.assertEqual(len(payable_move_lines.filtered(lambda l: l.reconciled)), 1)
 
-        register_pay2 = WizardRegister.create({
-            'journal_id': bank_journal.id,
-            'payment_method_id': outbound_pay_method.id,
-            'amount': 100,
-        })
-        register_pay2.expense_post_payment()
+        payment = self.env['account.payment.register']\
+            .with_context(active_model='account.move', active_ids=expense.account_move_id.ids)\
+            .create({'amount': 100.0})\
+            ._create_payments()
```

Comment:
berk... IMAO, it would be more elegant ot do
```python
        payment_obj = self.env['account.payment.register'].with_context(active_model='account.move', active_ids=expense.account_move_id.ids)
        payment = payment_obj.create({'amount': 100.0})._create_payments()
```